doc/source/reference/random/extending.rst

```diff
@@ -5,24 +5,23 @@
 Extending
 =========
 The BitGenerators have been designed to be extendable using standard tools for
-high-performance Python -- numba and Cython.  The `~Generator` object can also
+high-performance Python -- Numba and Cython.  The `~Generator` object can also
 be used with user-provided BitGenerators as long as these export a small set of
 required functions.
 
 Numba
 -----
-Numba can be used with either CTypes or CFFI.  The current iteration of the
-BitGenerators all export a small set of functions through both interfaces.
+Numba supports usage of `~Generator` object, alternatively users can use CFFI.
+The current iteration of the BitGenerators all export a small set of functions
```

Comment:
All the documentation refers to the current implementation, unless stated otherwise
```suggestion
BitGenerators all export a small set of functions
```